packages/flutter_tools/lib/src/windows/topaz_ofd_validator.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:process/process.dart';
+
+import '../base/io.dart';
+import '../doctor_validator.dart';
+
+const String kCoreProcessPattern = r'Topaz OFD\\Warsaw\\core\.exe';
+
+class TopazOfdValidator extends DoctorValidator {
```

Comment:
5 years from now we might forget why this code exists. Could we add a comment that links to https://github.com/flutter/flutter/issues/121366?

Reply:
Maybe we should call this MalwareScannerValidator, so that we can extend it in the future?